modulefiles/module_base.wcoss2.lua

```diff
@@ -26,16 +26,11 @@ load(pathJoin("prod_util", "2.0.9"))
 load(pathJoin("grib_util", "1.2.3"))
 load(pathJoin("bufr_dump", "1.0.0"))
 load(pathJoin("util_shared", "1.4.0"))
-load(pathJoin("crtm", "2.4.0"))
 load(pathJoin("g2tmpl", "1.9.1"))
+load(pathJoin("ncdiag", "1.0.0"))
+load(pathJoin("crtm", "2.4.0"))
 load(pathJoin("wgrib2", "2.0.7"))
 
-pushenv("HPC_OPT", "/apps/ops/para/libs")
-append_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/compiler/intel/19.1.3.304")
-append_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/mpi/intel/19.1.3.304/cray-mpich/8.1.7")
-
-load("ncdiag/1.0.0")
-
 prepend_path("MODULEPATH", pathJoin("/lfs/h2/emc/global/save/emc.global/git/prepobs/v1.0.1/modulefiles"))
```

Comment:
Not part of this PR but why do we pick up `prepobs/1.0.1` from a non-production space?

Reply:
We could certainly set this to use the officially installed production `prepobs/1.0.1` module on WCOSS2. Using a non-prod/development install provides a tad more flexibility though for development testing. Since this modulefile (like the others) is used for non-production/developer runs, I have us pointing to our own install. This is also consistent with what we do on the other platforms and is a historical thing (we used to point to non-prod installs of prepobs in `config.base` and still do for obsproc). When prepobs became a module we moved it from `config.base` to the modulefiles.

When we install newer prepobs versions in our own space, that may not be in production, we can update the version on this line. Once version files come into play in `develop` we'll just change the prepobs version in the version file and leave this line as is with a version variable. We will have a development version of prepobs in the near future to support the GFS post-com-reorg and won't be able to use the production module so we'll need the development install for a while.

Thinking ahead, when we plan a tag for ops that uses the production prepobs we can change this to use that (assuming it's already available and installed in production space). But this modulefile is not used in production anyway. By then we should have also broken the module_base files apart and set module loads per job, so this prepobs module load will have moved to wherever our development prep job sources its modules from.

Hope the above explains some of the history and reasoning in not picking up `prepobs` module from production space when on WCOSS2. Good question. :)

Reply:
Thank you for the detailed explanation.   It will be nice when module loads are at the job level.   This is consistent with operations.

Reply:
> It will be nice when module loads are at the job level. This is consistent with operations.

Agreed!